tests/benchmarks/micro_benchmarks/test_ib_traffic_performance.py

```diff
@@ -216,7 +216,24 @@ def test_ib_traffic_performance(self, mock_gpu):
 22798.8,23436.3
 23435.3,22766.5
 """
+# suppose gpu driver mismatch issue or other traffic issue cause -1 result
         raw_output_1 = """
```

Comment:
pls add the indent to the comment line to pass the lint

Reply:
Please pass the lint check